flink-connector-debezium/src/main/java/com/ververica/cdc/debezium/DebeziumSourceFunction.java

```diff
@@ -440,7 +440,11 @@ public void run(SourceContext<T> sourceContext) throws Exception {
                 (Gauge<Long>) () -> debeziumChangeFetcher.getEmitDelay());
         metricGroup.gauge(
                 "sourceIdleTime", (Gauge<Long>) () -> debeziumChangeFetcher.getIdleTime());
-
+        metricGroup.gauge(
```

Comment:
SourceReaderMetricConstants.PENDING_RECORDS ?

Reply:
`flink-connector-debezium` doesn't depend on `flink-cdc-base`, created `DebeziumSourceMetricConstants` to store string constants